src/richie/plugins/nesteditem/defaults.py

```diff
@@ -4,13 +4,13 @@
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
 
+ACCORDION, LIST = "accordion", "list"
```

Comment:
It's because I have to declare in the demo that I want an accordion, so I refer to this string via a variable to be DRY.
Like here: https://github.com/openfun/richie/pull/1016/files#diff-c6044ce4e87aec7c9b7977f97fe3f475R345

But you're right I didn't use it ^^ will fix now.